### DISCUSSION TITLE
gamemode.json.roomdata

### DISCUSSION THREAD

**Discussion by ZhenyaMartynov** [Sokol-Games-GmbH/PixelCombats.PublicApi#157] [Ideas]
Добавить возможность изменять данные при создании комнаты, например для этого сделать раздел roomdata.
Предлогаю свойства:
MapLists - тут все ясно
Parameters - тоже ясно
MaxPlayers - массив, в котором хранятся числа, минимальное 1, максимальное 24.
Суть в том что когда ты будешь выбирать максимальное количество игроков при создании комнаты, заместо стандартных настроек, тоесть 1,2,4,8итд там будет этот массив (MapPlayers)

**Comment by kkohno** [comment]
опишите подробнее? у нас есть такто возможность при мсоздании решима указать галочки, понятно что нужно еще цифры желательно. Список карт выбирает пользователь макс игроков тоже есть возможность задать, насколько помню, при разработке режима.

**Reply by Evgene82** [reply]
А что описать? Про то что можно задать макс.игроков - то нет

**Comment by kkohno** [comment]
понял, мысль... штук интересная, но скорее всего мы просто в режим завезем скрипты для конструктора комнат и Вы сможете запрограммировать поведение конструктора с его опциями
